scheduler/scheduler.go

```diff
@@ -194,6 +207,9 @@ func (r *Registry) newQueue(name string, c Config) *Queue {
 
 		q.activeRequestsMetricsKey = fmt.Sprintf("lifo.%s.active", name)
 		q.queuedRequestsMetricsKey = fmt.Sprintf("lifo.%s.queued", name)
+		q.rejectedRequestsMetricsKey = fmt.Sprintf("lifo.%s.rejected", name)
+		q.timedOutRequestsMetricsKey = fmt.Sprintf("lifo.%s.timed-out", name)
```

Comment:
Lets not use dash in the metric name here so it is single word and different metrics backed accidentally do not replace it with e.g. `_`

Reply:
Removed the `-`.